CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ function(packageProject)
     "DEPENDENCIES" ${ARGN}
   )
 
-  if(DEFINED PROJECT_NO_VERSION_SUFFIX)
+  if(PROJECT_NO_VERSION_SUFFIX)
```

Comment:
Good catch, should have checked the docs on single argument flags as well. TBH I'd actually prefer changing the flag to named argument, to be more consistent with the other syntax (e.g. `NO_VERSION_SUFFIX YES`). What do you think?

Reply:
but think, how it looks: `NO_VERSION_SUFFIX NO`
I would prefer it as an option, it seems clear to me.

Reply:
Yeah, the name should probably be changed as well, how about `DISABLE_VERSION_SUFFIX`. As an example, in

```cmake
packageProject(
  NAME ${PROJECT_NAME}
  [...]
  NO_VERSION_SUFFIX
)
```

vs


```cmake
packageProject(
  NAME ${PROJECT_NAME}
  [...]
  DISABLE_VERSION_SUFFIX YES
)
```

the latter seems more consistent to me.

Reply:
OK, but not today